src/Common/src/Abstractions/IApplicationInstanceInfo.cs

```diff
@@ -75,5 +75,5 @@ public interface IApplicationInstanceInfo
     /// <returns>
     /// The name of the application.
     /// </returns>
-    string ApplicationNameInContext(SteeltoeComponent steeltoeComponent, string additionalSearchPath = null);
+    string GetApplicationNameInContext(SteeltoeComponent steeltoeComponent, string additionalSearchPath = null);
```

Comment:
I prefer `GetApplicationNameInContext` too, because it reads more like a proper US English sentence. Though "context" in itself is a rather vague tem. How about `GetApplicationNameForComponent` instead?